config.go

```diff
@@ -90,6 +91,7 @@ type config struct {
 	PoolAddress       string  `long:"pooladdress" description:"The ticket pool address where ticket fees will go to"`
 	PoolFees          float64 `long:"poolfees" description:"The per-ticket fee mandated by the ticket pool, in coins"`
 	AddrIdxScanLen    int     `long:"addridxscanlen" description:"The width of the scan for last used addresses on wallet restore and start up (default: 750)"`
+	StakePoolFeeAddrs string  `long:"stakepoolfeeaddrs" description:"Enables the wallet as a stake pool with an extended key in the format of \"xpub...:index\" to derive cold wallet addresses to send fees to"`
```

Comment:
If it takes multiple it should be made consistent with the other options that take multiple entries in my opinion.

```Go
StakePoolFeeAddrs []string
```